### PR TITLE
Microsoft def adv paging bug

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/34245

## Description
added limitation to readme.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by bakatzir — COMMENTED** [review]
please add it to the command desc in the readme as well, and bump the version on a revision with `Documentation and metadata improvements.`

File: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_11.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Microsoft Defender for Endpoint
+- Documentation and metadata improvements.
```
Comment (bakatzir) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_11.md:4]:
```suggestion
- Documentation and metadata improvements.
- Upgraded the Docker image to: *demisto/crypto:1.0.0.16776*.
```

**Review by bakatzir — APPROVED** [review]

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml [hunk 2]
```diff
@@ -230,6 +230,7 @@ script:
       secret: false
     deprecated: false
     description: Retrieves a collection of machines that have communicated with WDATP cloud on the last 30 days.
+                 Due to an api limitation, maximum of 10,000 results can be returned.
```
Comment (amshamah419) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml:233]:
```suggestion
                 Due to a limitation of the API, a maximum of 10,000 results can be returned.
```

File: Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md [hunk 3]
```diff
@@ -19,6 +19,10 @@ For more details about the authentication used in this integration, see [Microso
 
 **Note**: If you previously configured the Windows Defender ATP integration, you need to perform the authentication flow again for this integration and enter the authentication parameters you receive when configuring the integration instance.
 
+## Limitations
+--
+Due to an api limitation, **microsoft-atp-get-machines** command can return maximum of 10,000 results.
```
Comment (amshamah419) [inline Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md:24]:
```suggestion
Due to a limitation of the API, the **microsoft-atp-get-machines** command can return a maximum of 10,000 results.
```

**Comment by orhovy** [comment]
Not relevant as a custom yml was made for the customer and works.

**Pull request closed without merging**